.gitignore

```diff
@@ -1,6 +1,7 @@
 *.iml
 .idea
 .gradle
+.kotlin
```

Comment:
Looks like an ignore specific for JVM projects
I've never seen `.kotlin` folder in an Android project